### PR TITLE
Fix issues with VB's ParseTrailingTrivia helper.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
tagging @dotnet/roslyn-compiler @RikkiGibson @cston @jcouv .

Small PR.  LMK what else you need here.  Found this in an IDE scenario where we were calling this to produce trailing trivia and we made trailing trivia inconsistent with what the parser produces.  Note that C# does the right thing here, i.e.: https://github.com/dotnet/roslyn/blob/4be125cab1f70b6af4ba7b476ee647eca975f3a3/src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs#L1611

**Comment by CyrusNajmabadi** [comment]
@dotnet/roslyn-compiler PTAL.

File: src/Compilers/VisualBasic/Portable/Syntax/SyntaxNodeFactories.vb [hunk 1]
```diff
@@ -114,7 +114,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Public Shared Function ParseTrailingTrivia(text As String, Optional offset As Integer = 0) As SyntaxTriviaList
             Dim s = New InternalSyntax.Scanner(MakeSourceText(text, offset), VisualBasicParseOptions.Default)
             Using s
-                Return New SyntaxTriviaList(Nothing, s.ScanMultilineTrivia().Node, 0, 0)
+                Return New SyntaxTriviaList(Nothing, s.ScanSingleLineTrivia().Node, 0, 0)
```
Comment (cston) [inline src/Compilers/VisualBasic/Portable/Syntax/SyntaxNodeFactories.vb]: >ScanSingleLineTrivia [](start = 55, length = 20)

Is this a change in behavior if the following line is a doc comment or a directive?

Comment (CyrusNajmabadi) [reply src/Compilers/VisualBasic/Portable/Syntax/SyntaxNodeFactories.vb]: Yes.  But that's intentional. This is the ParseTrailingTrivia helper. Those are not trailing trivia.

Comment (cston) [reply src/Compilers/VisualBasic/Portable/Syntax/SyntaxNodeFactories.vb]: Please consider adding tests for those cases so the intent is clear.

---
In reply to: [390483174](https://github.com/dotnet/roslyn/pull/42234#discussion_r390483174) [](ancestors = 390483174)

**Comment by CyrusNajmabadi** [comment]
Sure!

File: src/Compilers/VisualBasic/Portable/Syntax/SyntaxNodeFactories.vb (hunk 1 above)
Comment (CyrusNajmabadi) [reply src/Compilers/VisualBasic/Portable/Syntax/SyntaxNodeFactories.vb]: tests added!

**Comment by CyrusNajmabadi** [comment]
@dotnet/roslyn-compiler Another set of eyes please :)

**Comment by CyrusNajmabadi** [comment]
Thanks!

**Pull request merged**